test/Shared/Microsoft.DotNet.Scaffolding.Shared.Tests/StringUtilTests.cs

```diff
@@ -0,0 +1,40 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+
+namespace Microsoft.DotNet.Scaffolding.Shared.Tests
+{
+    public class StringUtilTests
+    {
+        [Fact]
+        public void ContainsIgnoreCase_ShouldReturnCorrectResults()
+        {
+            // Arrange
+            string input1 = "Hello";
```

Comment:
Add a case for when either value, or both, is `null`

Reply:
will add check and test case tyty